### PR TITLE
Theming from this weekend

### PR DESCRIPTION
**Pull request by amazingrando**
@generalredneck  This PR is all the work that I've done over the weekend. The needle is moving! I'll be doing more work on this probably late in the week.

### CODE REVIEW

File: config/system.performance.yml [hunk 1]
```diff
@@ -2,15 +2,15 @@ cache:
   page:
     max_age: 0
 css:
-  preprocess: true
+  preprocess: false
```
Comment (generalredneck) [inline config/system.performance.yml:5]: We will probably want this back on later

File: config/system.performance.yml [hunk 2]
```diff
@@ -2,15 +2,15 @@ cache:
...
   gzip: true
 fast_404:
   enabled: true
   paths: '/\.(?:txt|png|gif|jpe?g|css|js|ico|swf|flv|cgi|bat|pl|dll|exe|asp)$/i'
   exclude_paths: '/\/(?:styles|imagecache)\//'
   html: '<!DOCTYPE html><html><head><title>404 Not Found</title></head><body><h1>Not Found</h1><p>The requested URL "@path" was not found on this server.</p></body></html>'
 js:
-  preprocess: true
+  preprocess: false
```
Comment (generalredneck) [inline config/system.performance.yml:13]: Same here

**Review by generalredneck — APPROVED** [review]
So do a double check on the multidev and make sure the ststuff built correctly. I feel like it's missing stuff compared to the code you wrote but it is totally progress.

**Comment by amazingrando** [comment]
Sounds good. I did more work tonight that I will push up in the morning.

**Comment by amazingrando** [comment]
More work pushed up.

Still outstanding:
- [ ] App icon
- [ ] Favicon
- [ ] User login page

**Comment by generalredneck** [comment]
Just as a double check... something seems up because for sure I thought you moved the link around on the view... based on the code...
image

**Comment by amazingrando** [comment]
@generalredneck Something is going wrong. See screenshot below.

screenshot 2019-02-19 10 00 57

I've made config changes but I haven't done anything in the database on Pantheon.

**Comment by generalredneck** [comment]
Yeah the database should be automagically updating. I'll look into it!

**Comment by generalredneck** [comment]
@amazingrando So there is something up with that song book view... it doesn't pull in the configuration on the first go around for some reason... with that said, there are a couple of small issues still that I'm not able to quite figure out...
screenshot_2019-02-19_09-50-27

**Pull request merged**